src/main/kotlin/org/digma/intellij/plugin/ui/highlights/HighlightsMessageRouterHandler.kt

```diff
@@ -16,6 +17,7 @@ class HighlightsMessageRouterHandler(project: Project) : AbstractInsightsMessage
 
         when (action) {
             "HIGHLIGHTS/GET_PERFORMANCE" -> getHighlightsPerformance(browser, requestJsonNode)
+            "MAIN/GET_HIGHLIGHTS_TOP_ISSUES_DATA" -> getHighlightsTopInsights(browser, requestJsonNode)
```

Comment:
Let's change it to  "HIGHLIGHTS/GET_TOP_ISSUES_DATA" or I have to change my route to MAIN/GET_HIGHLIGHTS_PERFORMANCE_DATA